packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -143,6 +166,26 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
 
     const setSubmitForm = submitFormFn => (submitForm.current = submitFormFn);
 
+    React.useEffect(() => {
+        const disposer = reaction(
+            () => buy_sell_store.form_error_code,
+            () => {
+                if (buy_sell_store.form_error_code === api_error_codes.MARKET_RATE_CHANGE) {
+                    if (!isMobile()) {
+                        buy_sell_store.hidePopup();
+                        setTimeout(() => setShowMarketRateChangeErrorModal(true), 280);
```

Comment:
any reason why timeout is 280?

Reply:
there are some other modals with different timeout values, could we not choose one value for all?

Reply:
Yeapp in this PR https://github.com/binary-com/deriv-app/pull/6200 I declared a constant for modal duration in my profile store`MODAL_TRANSITION_DURATION = 250`, I'll add it here for now

Reply:
timeout duration should have been `250` instead of `280` my bad